src/server/unix.rs

```diff
@@ -250,16 +251,23 @@ pub fn instances_from_data_dir(dir: &Path, system: bool,
         if !item.file_type()?.is_dir() {
             continue;
         }
-        if let Some(name) = item.file_name().to_str() {
-            if !is_valid_name(name) {
-                continue;
-            }
-            instances.insert((name.to_owned(), system));
+
+        if let Some(name) = instance_name_from_os_str(&item.file_name()) {
```

Comment:
Why we are skipping `_localdev` here?

Reply:
Because `edb server` will use that directory, and it's not managed by the CLI. We skip `_localdev` here so that it doesn't show up in `edgedb instance status` as a local instance; instead `edb cli` will create a remote instance which will show up in the status list properly.

Reply:
Oh the creation is not a bug - `edb server` without `--data-dir` will just use that directory.